frontend/src/views/GShootItemPlaceholder.vue

```diff
@@ -151,6 +162,13 @@ export default {
           promises.push(this.ensureProjectTerminalShortcutsLoaded())
         }
         await Promise.all(promises)
+
+        if (routeName === 'ShootItemTerminal' && !this.isAdmin && !this.hasShootWorkerGroups) {
+          this.error = Object.assign(Error('Terminals are not avialble for this cluster'), {
+            code: 404,
+            reason: 'Shoot has no workers to schedule a terminal pod',
+          })
```

Comment:
```suggestion
          this.error = Object.assign(Error('Shoot has no workers to schedule a terminal pod'), {
            code: 404,
            reason: 'Terminal not available',
          })
```

<img width="631" alt="Screenshot 2023-09-20 at 16 39 57" src="https://github.com/gardener/dashboard/assets/5526658/254f2c2f-e4b5-48e7-befd-a5d91755f351">